ownCloud Intents/DeletePathItemIntentHandler.swift

```diff
@@ -22,9 +22,38 @@ import ownCloudSDK
 import ownCloudAppShared
 
 @available(iOS 13.0, *)
-public class DeletePathItemIntentHandler: NSObject, DeletePathItemIntentHandling {
+typealias DeletePathItemCompletionHandler = (DeletePathItemIntentResponse) -> Void
 
-	func handle(intent: DeletePathItemIntent, completion: @escaping (DeletePathItemIntentResponse) -> Void) {
+@available(iOS 13.0, *)
+public class DeletePathItemIntentHandler: NSObject, DeletePathItemIntentHandling, OCCoreDelegate {
+	weak var core : OCCore?
+	var completionHandler: DeletePathItemCompletionHandler?
+
+	func complete(with response: DeletePathItemIntentResponse) {
```

Comment:
This function is duplicated in some classes you changed (except of the parameter type). Does it makes sense to put this code into the superclass with the generic type INIntentResponse as parameter?

Reply:
Please see https://github.com/owncloud/ios-app/pull/930#discussion_r603624730.